extras/cronwrapper

```diff
@@ -51,11 +51,11 @@ if $LOGGING; then
     RET=1
   fi
 else
-  "${SCRIPT}" "${CONF}" 2>&1 >${LOGFILE}
+  "${SCRIPT}" "${CONF}" >${LOGFILE} 2>&1
```

Comment:
If you're not testing on a systemd machine you probably wouldn't have noticed either way since systemd is what does the actual printing to STDERR (not dpkg).